spec/lib/arclight/normalized_id_spec.rb

```diff
@@ -24,8 +24,6 @@
   context 'when the id is nil' do
     let(:id) { nil }
 
-    it 'returns an empty string' do
-      expect(normalized_id).to eq ''
-    end
+    it { expect { normalized_id }.to raise_error Arclight::Exceptions::IDNotFound }
```

Comment:
the coverage report says that the `message` isn't being tested here. you can add a second parameter that tests the message.